Libraries/Text/TextPropTypes.js

```diff
@@ -18,6 +18,15 @@ const TextStylePropTypes = require('TextStylePropTypes');
 
 const stylePropType = StyleSheetPropType(TextStylePropTypes);
 
+const DataDetectorTypes = [
```

Comment:
Chose dataDetectorTypes to keep it consistent with the existing property for TextInput. Should I change the prop name to LInkifyTypes or any variation of that?